ETHEREUM.md

```diff
@@ -16,60 +13,65 @@ incoming support funds, and others, translate into ethereum transactions.
 
 For users to approve those transactions originated in the app, they need to
 establish a connection between an Ethereum wallet and the app. We provide this
-capability through a WalletConnect integration. A substantial number of Ethereum
-wallets support WalletConnect. [You can find the complete list here][wcw].
+capability through a WalletConnect integration. A substantial number of
+Ethereum wallets support WalletConnect. You can find the complete list
+[here][wcw].
 
 Once a wallet is connected to the app, the funding experiences become available
-to the user. The user can now set up and edit their support, receive funds, etc.
-All of these actions translate into transactions the user must review, (i.e.,
-approve or reject) on their connected wallet.
+to the user. The user can now set up and edit their support, receive funds,
+etc.  All of these actions translate into transactions the user must review,
+(i.e., approve or reject) on their connected wallet.
 
-These same transactions are provided and ran by the [Radicle Contracts][rc], our
-custom Ethereum smart-contracts.
+These same transactions are provided and ran by the [Radicle Contracts][rc],
+our custom Ethereum smart-contracts.
 
-### Ethereum environments
+### Orgs
 
-The funding experiences can take place in three different environments:
 
-1. `Local`
+Radicle Orgs allow a group of people to manage multiple projects on Radicle.
 
-  In this environment, the Radicle Contracts are deployed to development Ethereum
-  node running locally and we use a local test wallet that can communicate with it.
-  The "DAI" token used in this environment is our custom Radicle Token. See the
-  [local environment](#local-environment) section to learn how to get set up.
+What’s cool about Orgs is that they’re actually smart contracts that can be
+managed by DAOs or multi-sigs. With Radicle Orgs, projects can maintain an
+auditable and transparent history of project state on-chain. This enables new
```

Comment:
A bit less editorializing ;) and swapping the what and how:

> Radicle orgs allow users to maintain an auditable and transparent history of project state on Ethereum. [...] Org contracts may be controlled by arbitrary addresses. At the moment we deploy a Gnosis safe contract (link) as the controller when creating an org.

Reply:
👍 I just copy-pasted it straight from https://radicle.community/t/feature-update-orgs/2132/4.

Reply:
c87333c01e30930a8771878ac5f552e3bb4609f7